draft-ietf-anima-constrained-join-proxy.md

```diff
@@ -151,48 +163,67 @@ Registrar/Coordinator (JRC), Pledge, and Voucher.
 In this document, the term "Registrar" is used throughout instead of "Join
 Registrar/Coordinator (JRC)".
 
-The term "installation network" refers to all devices in the installation and the network connections between them. The term "installation IP_address" refers to an address out of the set of addresses which are routable over the whole installation network.
+The term "installation" refers to all devices in the network and their interconnections, including Registrar, enrolled nodes with and without constrained Join Proxy functionality and Pledges.
+
+(Installation) IP addresses are assumed to be routeable over the whole installation network except for link-local IP addresses.
 
 The "Constrained Join Proxy" enables a pledge that is multiple hops away from the Registrar, to securely execute the BRSKI protocol {{RFC8995}} over a secure channel.
 
 The term "join Proxy" is used interchangeably with the term "constrained Join Proxy" throughout this document.
 
+The {{RFC8995}} Circuit Proxy is referred to as a TCP circuit Join Proxy.
+
 # Requirements Language {#reqlang}
 
 {::boilerplate bcp14}
 
 # constrained Join Proxy functionality
 
-As depicted in the {{fig-net}}, the Pledge (P), in a Low-Power and Lossy Network (LLN) mesh
+As depicted in the {{fig-net}}, the Pledge (P), in a network such as a Low-Power and Lossy Network (LLN) mesh
  {{RFC7102}} can be more than one hop away from the Registrar (R) and not yet authenticated into the network.
 
 In this situation, the Pledge can only communicate one-hop to its nearest neighbor, the constrained Join Proxy (J) using their link-local IPv6 addresses.
 However, the Pledge needs to communicate with end-to-end security with a Registrar to authenticate and get the relevant system/network parameters.
-If the Pledge, knowing the IP-address of the Registrar, initiates a DTLS connection to the Registrar, then the packets are dropped at the constrained Join Proxy since the Pledge is not yet admitted to the network or there is no IP routability to Pledge for any returned messages from the Registrar.
+If the Pledge, knowing the IP-address of the Registrar, initiates a DTLS connection to the Registrar, then the packets are dropped at the constrained Join Proxy since the Pledge is not yet admitted to the network or there is no IP routability to the Pledge for any returned messages from the Registrar.
 
 ~~~~ aasvg
                     multi-hop mesh
-         .---.
-         | R +---.    +----+    +---+        +--+
+         .---.                         IPv6
```

Comment:
The term 'IPv6 subnet' inserted here is not explained in the text. And it may raise some questions. The entire mesh is a multi-link IPv6 subnet but this isn't shown by text, for simplicity.  Do we want to say that the link between JP and Pledge is also a (different) IPv6 subnet? One where only link-local addresses are valid. Does it help to point this out?